src/ConnectedKubernetes/custom/New-AzConnectedKubernetes.ps1

```diff
@@ -149,8 +187,8 @@ function New-AzConnectedKubernetes {
         $PSBoundParameters.Add('IdentityType', $IdentityType)
 
         #Region check helm install
+        $HelmVersion = helm version --short --kubeconfig $KubeConfig
         try {
-            $HelmVersion = helm version --short --kubeconfig $KubeConfig
```

Comment:
Why move this line out of try block?

Reply:
In order to be able to use this variable in another method block(Line 233, 248), I moved it out, which now seems dangerous.
I will correct the error and retrieve its value in another method block.